package/environments/development.js

```diff
@@ -42,6 +42,7 @@ if (
       headers: devServer.headers,
       overlay: devServer.overlay,
       stats: {
+        colors: true,
```

Comment:
This is different from the value that may be generated by `webpack-dev-server`, which is using `supports-color` to determine the default value for this option

on maxOS, using webpacker 5, the colors option looks like: 
```
    colors: { level: 2, hasBasic: true, has256: true, has16m: false }
```

Reply:
Thanks for the tip, that does look like a more flexible option.

Reply:
Hi @rossta,

Apologies, I'm not a JavaScript developer.

This will change anyway in webpack-dev-server 4: they have removed [supports-color](https://www.npmjs.com/package/supports-color) in the PR you mentioned at https://github.com/rails/webpacker/issues/2936#issuecomment-785359018, but I did not get the reason

I can confirm that launching `webpack-dev-server` or `webpack serve` with `webpack-dev-server@next` sets the `colors` option to `true`:

```
yarn webpack --version
yarn run v1.22.10
webpack 5.24.1
webpack-cli 4.5.0
webpack-dev-server 4.0.0-beta.0
```

```js
    stats: { preset: 'normal', colors: true },
```

Reply:
Good to hear. Looks from the [webpack-dev-server 4.0.0-beta.0 release notes](https://github.com/webpack/webpack-dev-server/releases/tag/v4.0.0-beta.0) that the `stats` option is removed, so these `devServer` options won't be needed. 

> the stats option was removed, please use the stats option from webpack.config.js

I'll await feedback from the maintainers who could either merge this PR as a temporary improvement or close it in favor of the future webpack-dev-server v4 integration (in which case, the workaround noted in #2936 will work in the mean time).